packages/venia-concept/src/components/CreateAccount/createAccount.js

```diff
@@ -43,25 +47,28 @@ class CreateAccount extends Component {
     get hasEmailError() {
         // Only return true if the email field has a value, and it's not being checked against existing emails
         return (
-            !!this.state.email &&
+            !!this.getFieldValue(fields.email) &&
             !this.state.emailAvailable &&
             !this.state.checkingEmail
         );
     }
 
     get hasPasswordConfirmError() {
         // Check if passwords match
-        return this.state.password !== this.state.passwordConfirm;
+        return (
+            this.getFieldValue(fields.password) !==
+            this.getFieldValue(fields.confirmPassword)
+        );
     }
 
     get isIncompleteOrInvalid() {
         // A catch-all to check if the form is valid so the customer can attempt to create an account
         return (
             this.hasEmailError ||
             this.hasPasswordConfirmError ||
-            !this.state.email ||
-            !this.state.firstName ||
-            !this.state.lastName
+            !this.getFieldValue(fields.email) ||
```

Comment:
It'd be great if all of this was checked through `Informed`. Separate issue probably.